### PR TITLE
Rishavmehra/improvement

### PR DESCRIPTION
**Pull request by rishavmehra**
Issue: [Add existing applications to CI linting process](https://github.com/keploy/keploy/issues/2317)

Summary of Changes:
- Integrated existing applications into the CI linting process
- Implemented a code fix for the http-pokeapi application
- Added a new build pipeline

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+name: Build Go Sample Projects
+
+on:
+  push:
+    branches:
+      - master
```
Comment (khareyash05) [inline .github/workflows/build.yml:6]: main would suffice

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      - main
+  pull_request:
+    types: [opened, edited, synchronize, reopened]
```
Comment (khareyash05) [inline .github/workflows/build.yml:9]: not needed

**Review by khareyash05 — COMMENTED** [review]
Please address the comments

**Comment by rishavmehra** [comment]
@khareyash05 issue fixed

**Review by khareyash05 — APPROVED** [review]

**Pull request merged**